lstchain/paths.py

```diff
@@ -26,7 +26,7 @@
     r'LST-(\d+)'         # tel_id
     r'(?:.(\d+))?'       # stream is optional
     r'.Run(\d+)'         # run number
-    r'.(\d+)'            # subrun number
+    r'(?:.(\d+))?'            # subrun number
```

Comment:
Are these question marks added to make subrun number optional?

Reply:
Yes. For when we use merged DL1/2 files

Reply:
The first question mark results in the group being "non-capturing". This prohibits accessing it and should be removed. The second question mark makes it optional